changelog/README.md

```diff
@@ -0,0 +1,34 @@
+How to add a new changelog entry to the pending changelog?
+==========================================================
+
+This will get copied to dlang.org and cleared when master gets
+merged into stable prior to a new release.
+
+```
```

Comment:
I'm just trying to add the changelog for druntime PR and I find this very confusing. Is what follows the content of the file? If so, what's the name of the file? Maybe add more documentation around here.